### PR TITLE
feat(pino): change bunyan to pino

### PR DESCRIPTION
**Pull request by OlivierCuyp**
add more env vars config

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,81 +1,13 @@
...
-  level: config.logLevel || 'info',
-  serializers: {
-    error: bunyan.stdSerializers.err
-  },
-  streams: [bunyanStream]
+const Logger = require('./libs/Logger');
+const createUdpStream = require('./helpers/createUdpStream');
+const env = process.env;
+const destination = env.LOG_ENDPOINT
+  ? createUdpStream(env.LOG_ENDPOINT)
```
Comment (SagaStalin) [inline src/index.js]: INFO Convert this usage of the ternary operator to an "if"/"else" structure. [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AConditionalOperator)

File: src/libs/Logger.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        projectName,
+        buildNumber,
+        commit
+      },
+      level: logLevel || 'info',
+      timestamp: () => `,"time":"${new Date().toISOString()}"`,
+      serializers: {
+        error: pino.stdSerializers.err
+      },
+      prettyPrint: pretty ? { colorize: true } : false
```
Comment (SagaStalin) [inline src/libs/Logger.js]: INFO Convert this usage of the ternary operator to an "if"/"else" structure. [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AConditionalOperator)

File: src/libs/Logger.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    }, destination);
+
+    this.stackLevelIndex = levels.indexOf(stackLevel);
+  }
+
+  _buildLog (logLevel, event, data = null, meta = null) {
```
Comment (SagaStalin) [inline src/libs/Logger.js]: MINOR Rename this '_buildLog' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS100)

File: src/helpers/createUdpStream.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+const dgram = require('dgram');
+const { Writable } = require('stream');
+
+const createUdpStream = (endpoint) => {
```
Comment (SagaStalin) [inline src/helpers/createUdpStream.js]: INFO Remove parentheses around the parameter of this arrow function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3524)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 1 minor
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by OlivierCuyp** [comment]
[2(error-logging): Improve error logging](https://trello.com/c/z9DDJqij/3795-2error-logging-improve-error-logging)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 1 minor
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SagaStalin** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**